tests/test_feature_extraction_beit.py

```diff
@@ -76,6 +77,26 @@ def prepare_feat_extract_dict(self):
         }
 
 
+def prepare_semantic_single_inputs():
+    ds = load_dataset("hf-internal-testing/fixtures_ade20k", split="test")
```

Comment:
Let's give a better name than just `ds`, we try to avoid such short names in the codebase.